test/cpp/api/serialize.cpp

```diff
@@ -218,15 +218,15 @@ TEST(SerializeTest, KeysFunc) {
   torch::serialize::OutputArchive output_archive;
   for (const auto i : c10::irange(3)) {
     output_archive.write(
-        "element/" + c10::to_string(i), c10::IValue(static_cast<int64_t>(i)));
+        "element/" + std::to_string(i), c10::IValue(static_cast<int64_t>(i)));
```

Comment:
There is also a fmt::to_string, but it's not in the standard fmt header

Reply:
I would use fmt::format here

Reply:
Ah, makes sense.